external-workshops/using-training-materials.md

```diff
@@ -0,0 +1,118 @@
+# Using Data Lab Training Materials
+
+[All Data Lab training materials](https://github.com/AlexsLemonade/training-modules/tree/master), including lecture slides, reference documents, instruction notebooks, and exercise notebooks are [openly licensed](https://github.com/AlexsLemonade/training-modules/blob/master/LICENSE.md), modifiable, and freely available to download via GitHub. 
+Whether you are instructing in-person or virtually, you will need access to all of the materials that apply to your workshop topic and infrastructure for hosting the materials. 
+
+**All workshop participants must sign the Data Lab’s training waiver before participating in a workshop that uses our materials. 
+[Participants can access and sign the waiver using this link](https://app.hellosign.com/s/2o2yghCj).**
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+* [Data Lab RStudio Server](#data-lab-rstudio-server)
+	* [How the Data Lab uses the RStudio server](#how-the-data-lab-uses-the-rstudio-server)
+	* [Using the Data Lab RStudio server for your workshop](#using-the-data-lab-rstudio-server-for-your-workshop)
+* [Using your own infrastructure](#using-your-own-infrastructure)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Data Lab RStudio Server
+
+### How the Data Lab uses the RStudio Server
+
+For both in-person and virtual workshops, we utilize [our RStudio server](https://rstudio.ccdatalab.org/auth-sign-in?appUri=%2F).
+For each workshop, we set up accounts and home directories for participants and pre-install the tools (R packages, command line tools), data (raw and processed files), and notebook files that will be used during the workshop. 
+Instructors can live code and walk participants through each notebook in real-time. 
+
+Each participant will be able to log in to their account using their own web browser, which gives them full access to all pre-loaded materials. 
+Once logged in, participants can code alongside instructors and complete their own copy of each exercise notebook.
+The Data Lab allows training participants to have access to our RStudio server for 6 months after the workshop ends.
+
+### Using the Data Lab RStudio server for your workshop
+
+**Training materials** 
+
+**1.) If you plan to modify any training materials, you must download our [`training-modules`](https://github.com/AlexsLemonade/training-modules/tree/master) repository or fork the repository on GitHub.** 
```

Comment:
I'd also rephrase this a bit more generally to say, if you want to modify you need to get your own copy of materials which you can do either by downloading ZIP or forking.